src/constants/shared.js

```diff
@@ -2,5 +2,7 @@ export default {
   BASE_URL: process.env.BASE_URL || '',
   EXTENSION: '.mp3',
   DOWNLOAD_DIR: 'tmp',
-  BUCKET_NAME: 'better-zoo'
+  BUCKET_NAME: 'better-zoo',
+  MAX_ATTEMPTS: 3,
```

Comment:
use `process.env.MAX_ATTEMPTS || 3` (as well with the retry_interval) so we can set these settings via ENV variables (as this is hosted in heroku)